model/model.js

```diff
@@ -28,3 +29,22 @@ exports.selectArticles = () => {
       return rows;
     });
 };
+
+exports.selectArticleById = (article_id) => {
+  return checkIfValidId(article_id)
```

Comment:
I like what you're thinking here, but PSQL will actually return automatic errors from invalid data types for you